configuration/etl/etl.d/xdmod-migration-9_5_0-10_0_0.json

```
@@ -203,39 +220,7 @@
                      "schema": "modw_aggregates"
                  }
              }
-          },
-          {
-            "name": "update-schema-version-history-tables-to-innodb",
-            "description": "Update modw_cloud tables to InnoDB",
-            "class": "ManageTables",
-            "definition_file_list": [
-                "common/hpcdb/schema-version-history.json"
-            ],
-            "endpoints": {
-                "destination": {
-                    "type": "mysql",
-                    "name": "XDMoD Data Warehouse",
-                    "config": "datawarehouse",
-                    "schema": "mod_hpcdb"
-                }
-            }
-        },
-        {
-            "name": "update-shredder-tables-to-innodb",
-            "description": "Update modw_cloud tables to InnoDB",
-            "class": "ManageTables",
-            "definition_file_list": [
-                "common/shredder/schema-version-history.json"
-            ],
-            "endpoints": {
-                "destination": {
-                    "type": "mysql",
-                    "name": "XDMoD Data Warehouse",
-                    "config": "datawarehouse",
-                    "schema": "mod_shredder"
-                }
-            }
-        },
+         },
```

```suggestion
        },
```

More indentation that shouldn't change.